docs/sources/setup-grafana/configure-security/audit-grafana.md

```diff
@@ -326,6 +326,17 @@ external group.
 | Set licensing token      | `{"action": "create", "requestUri": "/api/licensing/token"}` |
 | Save billing information | `{"action": "billing-information"}`                          |
 
+#### Cloud migration management\*
```

Comment:
why not just put a statement here that this feature requires a feature toggle, list the feature toggle, and link to the feature toggle page. 

Is this feature experimental, private or public preview? If so, you can use one of the standard admonitions in the shortcodes. https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsprivate-preview

Reply:
What about this? Do you want to also list the feature toggle?
```suggestion
#### Cloud migration management
 
Cloud migration management is available in Grafana Enterprise for users using the Cloud Migration Assistant.
You must activate a feature toggle to access this feature. 
  
```

Reply:
<img width="801" alt="image" src="https://github.com/user-attachments/assets/d17b12bb-dc7d-43e6-942c-43f709432c1d">

Thanks so much for looking into this 🙏  I tried your suggestion and this is how it looks. I think just the public preview annotation is sufficient, but can leave both the text and the note. What do you think?

Reply:
This looks great. The public preview annotation is enough. I had written the example before putting in the admonition.